src/System.Private.ServiceModel/tests/Common/Scenarios/project.json

```diff
@@ -1,10 +1,14 @@
 {
   "dependencies": {
     "Microsoft.NETCore.Platforms": "1.0.1-rc3-23809",
-    "System.Collections.Concurrent": "4.0.0",
-    "System.Net.Http": "4.0.1-rc3-23809",
+	"System.Collections.Concurrent": "4.0.0",
+	"System.ComponentModel.EventBasedAsync": "4.0.10",
+	"System.Net.Http": "4.0.1-rc3-23809",
```

Comment:
It probably doesn't need it, I was just updating versions based on warnings, I was only fixing the indent in this file, I'll change the version of S.N.Http when I rebase to match what I did for S.P.SM if nothing else just to be consistent.